tests/opkg.py

```diff
@@ -36,7 +36,8 @@ def runTest(self):
 class OpkgUpdate(rootfs_boot.RootFSBootTest):
     '''Opkg is able to update list of packages.'''
     def runTest(self):
-        board.sendline('\nopkg update && echo "All package lists updated"')
+        board.sendline("\nopkg update | tee opkg.txt && ! grep 'Signature check failed' opkg.txt"
+                        "&& echo 'All package lists updated' && rm opkg.txt")
```

Comment:
I think opkg.txt will get removed only if all subcommands runs succesfully. I would send separately the command:
```python
board.sendline("\nrm opkg.txt")
```
You should also remove it in ```recover``` function.